src/Confluent.Kafka/Confluent.Kafka.csproj

```diff
@@ -13,7 +13,7 @@
     <Title>Confluent.Kafka</Title>
     <AssemblyName>Confluent.Kafka</AssemblyName>
     <VersionPrefix>0.11.1-RC1</VersionPrefix>
-    <TargetFrameworks>net45;net46;netstandard1.3</TargetFrameworks>
```

Comment:
Lot of peoples are still under net45 I think (unfortunately), and having confluent.Kafka available in net45 allowed us to deploy some kafka logic without having to rewrite all our apps or make an upgrade (of all prods machine, which is not something companies will be eager to do... perhaps do some poll on #clients in slack and/or here to get some feedback? )

Reply:
yes, taking these out is a temporary convenience (for intellisense in vs code on mac/linux + build) - don't look too closely at this pr yet, just the general gist of it..